include/selene/Registry.h

```diff
@@ -78,5 +80,15 @@ class Registry {
             {_state, _metatables, name, funs...});
         _classes.push_back(std::move(tmp));
     }
+
+    void SetExceptionHandler(exception_handler handler) {
```

Comment:
We are copying the exception handler in correct? Do you mean `exception_handler&&` here?